README.md

```diff
@@ -4,10 +4,11 @@ See documentation [here](doc/00-overview.md)
 
 ``` yaml
 use-extension:
-  "@autorest/cli.common": "latest"
+  "@autorest/cli.common": "/home/qiaozha/code/autorest.cli.common"

```

Comment:
seems it's your local path

Reply:
yes, but cli.common has not released yet. can you release it first